cli/client/util.go

```diff
@@ -0,0 +1,17 @@
+package client
+
+import "context"
+
+type ContextKey string
+
+const (
+	Hosted = ContextKey("hosted")
+)
+
+func IsHosted(ctx context.Context) bool {
```

Comment:
Remove this file - it unnecessarily complicates things

Reply:
we need as a common file  between runner and lite engine

Reply:
removed